setup.py

```diff
@@ -61,6 +61,7 @@ def version_scheme(version):
             'sshuttle = sshuttle.cmdline:main',
         ],
     },
+    python_requires='>=3',
```

Comment:
putting the entry as `python_requires='>=3.5'` works too, so I'll leave it up to maintainers on what to restrict.

Reply:
Maybe that might be better, I think we do require 3.5